server/properties.lua

```diff
@@ -1 +1,540 @@
+end)
+
+--- Check for expired rents
+local function PropertiesRentCheck()
+    local rentedproperties = MySQL.query.await('SELECT * FROM properties WHERE NOT rent_expiration = false', {})
+    if not rentedproperties then return end
+
+    for _, v in pairs(rentedproperties) do
+        if v.rent_expiration / 1000 > os.time() or os.date("%x", v.rent_expiration / 1000) == os.date("%x", os.time()) then goto continue end
+        MySQL.Async.execute('UPDATE properties SET rent_expiration = NULL, garage_slots = NULL WHERE id = ?', { v.id })
+        MySQL.Async.execute('DELETE FROM property_owners WHERE property_id = ?', { v.id })
+
+        local renters = MySQL.query.await('SELECT citizenid FROM property_owners WHERE property_id = ? AND role = 1', { v.id })
+        if not renters then goto continue end
+        for _, owner in pairs(renters) do
+            TriggerEvent('qb-phone:server:sendNewMailToOffline', owner.citizenid, {
+                sender = 'Real Estate Agency',
+                subject = 'Rent Expired',
+                message = 'The rent of '.. v.name ..' expired.',
+            })
+        end
+        ::continue::
+    end
+end
+
+local function addPropertyToList(propertyData, propertyId)
+    local property = formatPropertyData(propertyData, {})
+    properties[propertyId] = property
+    updatePropertiesGroups()
+end
+
+RegisterNetEvent('qbx-properties:server:CreateProperty', function(propertyData)
+    if not propertyData then return end
+    local source = source
+
+    local player = QBCore.Functions.GetPlayer(source)
+    if not player then return end
+
+    if player.PlayerData.job.type ~= 'realestate' then return end
+
+    local propertyId = createProperty(propertyData)
+    if not propertyId then
+        QBCore.Functions.Notify(source, Lang:t('error.failed_createproperty'), 'error')
+        return
+    end
+
+    propertyData.name = propertyId .. ' ' .. propertyData.name
+    addPropertyToList(propertyData, propertyId)
+    TriggerClientEvent('qbx-properties:client:refreshProperties', -1)
+end)
+
+local function hasMoney(player, amount)
+    if not player then return false end
+    if player.Functions.GetMoney('cash') >= amount then
+        return 'cash'
+    end
+    if player.Functions.GetMoney('bank') >= amount then
+        return 'bank'
+    end
+    return false
+end
+
+--- Sets the role of a player for a property
+--- @param playerId integer
+--- @param propertyId integer
+--- @param role string
+--- @return boolean
+local function setRole(playerId, propertyId, role)
+    if not playerId or not role or not propertyId then return false end
+    local player = QBCore.Functions.GetPlayer(playerId)
+    if not player then return false end
+    local result = MySQL.insert.await('INSERT INTO property_owners (`property_id`, `citizenid`, `role`) VALUES (?, ?, ?)', {
+        propertyId, player.PlayerData.citizenid, role
+    })
+    return result and true or false
```

Comment:
If you really want it to return a boolean use `not not result` instead of something unnecessary